.github/workflows/pipeline.yml

```
@@ -0,0 +1,51 @@
+        branches:
+        - main
+    pull_request:
+        branches:
+        - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 4

(indentation)